resources/views/recurrings/index.blade.php

```diff
@@ -9,7 +9,7 @@
                 <h2>{{ __('models.recurrings') }}</h2>
             </div>
             <div class="row__column row__column--compact row__column--middle">
-                <a href="/recurrings/create" class="button">{{ __('actions.create') }} {{ __('models.recurring') }}</a>
+                <a href="/transactions/create" class="button">{{ __('actions.create') }} {{ __('models.recurring') }}</a>
```

Comment:
Why are we changing it to transactions? We're making a recurring entry, which in turn makes a transaction. Right?

Reply:
Because I think for uniformity, use the TransactionWizard, so therefore use transactions/create. I don’t like the recurring/create-page

Reply:
And because that recurring/create-page can only create a recurring spending and no recurring earning